lib/portage/tests/resolver/ResolverPlayground.py

```diff
@@ -683,7 +684,7 @@ def _load_config(self):
             create_trees_kwargs["target_root"] = self.target_root
 
         env = {
-            "PATH": os.environ["PATH"],
+            "PATH": f"{self.eprefix}/usr/sbin:{self.eprefix}/usr/bin:{os.environ['PATH']}",
```

Comment:
Do the hacks from e.g. `lib/portage/tests/emerge/test_libc_dep_inject.py`, `lib/portage/tests/emerge/test_emerge_blocker_file_collision.py` need adjusting too?

Reply:
Yeah, updated other things using ResolverPlayground to inhert its PATH.

Reply:
Thanks!